src/main/java/io/confluent/connect/jdbc/source/TimestampIncrementingTableQuerier.java

```diff
@@ -171,9 +171,9 @@ public SourceRecord extractRecord() throws SQLException {
     final Map<String, String> partition;
     switch (mode) {
       case TABLE:
-        String name = tableId.tableName(); // backward compatible
-        partition = Collections.singletonMap(JdbcSourceConnectorConstants.TABLE_NAME_KEY, name);
-        topic = topicPrefix + name;
+        String name = tableId.tableName();
+        topic = topicPrefix + name;// backward compatible
+        partition = OffsetProtocols.sourcePartitionForProtocolV1(tableId);
```

Comment:
Isn't this constant for this querier instance? Can we initialize this once in the constructor and then simply reuse the same map over and over?